mass_editing/tests/test_mass_editing.py

```diff
@@ -16,6 +16,9 @@ def setUp(self):
         self.mass_wiz_obj = self.env['mass.editing.wizard']
         self.mass_object_model = self.env['mass.object']
         self.res_partner_model = self.env['res.partner']
+        self.ir_translation_model = self.env['ir.translation']
+        self.lang_model = self.env['res.lang']
+
```

Comment:
Don't add empty extra lines in the middle of a method.

Reply:
Why not? We separate model connections, to the logic itself. IMHO it improves legibility.

Reply:
No, it only adds unneeded lines. PEP8 recommends not to put them. If you want to add a line with a comment, OK. That's the way to separate things inside the same method. Adding empty lines, not.

Reply:
I already removed the empty lines. Thanks for the suggestion